collector/aws/elb.go

```diff
@@ -201,27 +201,15 @@ func (el *ELBManager) GetPricingFilterInput() *pricing.GetProductsInput {
 	return &pricing.GetProductsInput{
 		ServiceCode: &el.servicePricingCode,
 		Filters: []*pricing.Filter{
-
-			&pricing.Filter{
-				Type:  awsClient.String("TERM_MATCH"),
-				Field: awsClient.String("usagetype"),
-				Value: awsClient.String("LoadBalancerUsage"),
-			},
-			&pricing.Filter{
+			{
 				Type:  awsClient.String("TERM_MATCH"),
 				Field: awsClient.String("productFamily"),
-				Value: awsClient.String("Load Balancer-Application"),
+				Value: awsClient.String("Load Balancer"),
 			},
-			&pricing.Filter{
-				Type:  awsClient.String("TERM_MATCH"),
-				Field: awsClient.String("TermType"),
-				Value: awsClient.String("OnDemand"),
-			},
-
-			&pricing.Filter{
+			{
 				Type:  awsClient.String("TERM_MATCH"),
-				Field: awsClient.String("group"),
-				Value: awsClient.String("ELB:Balancer"),
+				Field: awsClient.String("groupDescription"),
+				Value: awsClient.String("LoadBalancer hourly usage by Classic Load Balancer"),
```

Comment:
There is not better way to filter the price? 
I not like the way that we are filter by `*description` field